eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         inputs:
           packageType: sdk
           version: "$(DotNetCoreSDKVersion)"
-      - script: "dotnet test ${{parameters.ProjectFile}} --filter TestCategory!=Live --logger trx"
+      - script: "dotnet test eng/service.proj --filter TestCategory!=Live --logger trx /p:ServiceDirectory=${{parameters.ServiceDirectory}}"
```

Comment:
Does this prevent us from building all? I guess we can always pass `*` in for the ServiceDirectory property to build them all. At least until we decide we don't want to build everything.